Create rule S6317: AWS IAM policies should not allow privilege escalation (#1284)
SonarSource/rspec: rules/S6317/description.adoc

|Create a new access key for any user

|Create Login Profile

> |Create a login profile with a chosen password for a user the attacker controls

Reviewer: Here the attacker controls the password and then gets access to the user. From what I understand, controlling the user is not a prerequisite, it's the end goal.

> |Create a login profile with a chosen password for a user the attacker controls

Reviewer: ```suggestion
|Create a login profile with a password chosen by the attacker
```

|Update Login Profile 

> |Update the existing password of a user the attacker controls

Reviewer: ```suggestion
|Update the existing password with one chosen by the attacker
```
